node/src/domain/entity/Habit.ts

```diff
@@ -0,0 +1,13 @@
+import { DailyHabit } from "./DailyHabit"
+
+export class Habit {
+  public name!: string
+  public dailiesHabits: DailyHabit[] = []
+
+  constructor(name: string) {
+    this.name = name
+  }
+  addDailyHabit(dailyHabit: DailyHabit) {
```

Comment:
```suggestion
  }
  
  addDailyHabit(dailyHabit: DailyHabit) {
```

Reply:
I believe that you can setting the habit in daily habit here and remove setting DailyHabit constructor.